sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -1906,8 +1906,8 @@ case class ArrayPosition(left: Expression, right: Expression)
 @ExpressionDescription(
   usage = """
     _FUNC_(array, index) - Returns element of array at given (1-based) index. If index < 0,
-      accesses elements from the last to the first. Returns NULL if the index exceeds the length
-      of the array.
+      accesses elements from the last to the first. If the index exceeds the length of the array,
+      Returns NULL if ANSI mode is off; Throws ArrayIndexOutOfBoundsException when ANSI mode is on.
```

Comment:
How about rewriting it like this?
```
 The function returns NULL if the index exceeds the length of the array and `spark.sql.ansi.enabled` is set to false. If `spark.sql.ansi.enabled` is set to true, it throws ArrayIndexOutOfBoundsException for invalid indices.
```
by referring to the `Size` usage:
https://github.com/apache/spark/blob/8760032f4f7e1ef36fee6afc45923d3826ef14fc/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala#L79-L82